### PR TITLE
block ID description correction

### LINKED ISSUE #172: [docs] fix incorrect Block ID description
File: [docs/04_protocol/03_network_peer_protocol.md](https://github.com/EOSIO/welcome/blob/develop/docs/04_protocol/03_network_peer_protocol.md)

3.1. Block ID
The most significant 32 bits of the hash are retained while the least significant 32 bits are assigned the block number.
==>
The most significant 32 bits are assigned the block number while the least significant 224 bits of hash are retained.

### PR DESCRIPTION
**Pull request by bobgt**
Resolves #172 

Updated the description for block ID based on received feedback. 

@taokayan validated the feedback.

### CODE REVIEW

File: docs/04_protocol/03_network_peer_protocol.md [hunk 1]
```diff
@@ -888,4 +888,4 @@ Protocol messages are placed in a buffer queue and sent to the appropriate conne
 
 # 5. Protocol Improvements
 
-Any software updates to the p2p protocol must also scale progressively and consistently across all nodes. This translates into installing updates that reduce operation downtime and potentially minimize it altogether while deploying new functionality in a backward compatible manner, if possible. On the other hand, data throughput can be increased by taking measures that minimize message footprint, such as using data compression and binary encoding of the protocol messages.
\ No newline at end of file
+Any software updates to the p2p protocol must also scale progressively and consistently across all nodes. This translates into installing updates that reduce operation downtime and potentially minimize it altogether while deploying new functionality in a backward compatible manner, if possible. On the other hand, data throughput can be increased by taking measures that minimize message footprint, such as using data compression and binary encoding of the protocol messages.
```
Comment (iamveritas) [inline docs/04_protocol/03_network_peer_protocol.md]: what is different here?

File: docs/04_protocol/03_network_peer_protocol.md [hunk 2]
```diff
@@ -504,7 +504,7 @@ The operation mode for each node is stored in a sync manager context within the
...
+The EOSIO software checks whether two blocks match or hold the same content by comparing their block IDs. A block ID is a function that depends on the contents of the block header and the block number (see [Consensus Protocol: 5.1. Block Structure](01_consensus_protocol.md#51-block-structure)). Checking whether two blocks are equal is crucial for syncing a node’s local chain with that of its peers. To generate the block ID from the block contents, the block header is serialized and a SHA-256 digest is created. The most significant 32 bits are assigned the block number while the least significant 224 bits of hash are retained. Note that the block header includes the root hash of both the transaction merkle tree and the action merkle tree. Therefore, the block ID depends on all transactions included in the block as well as all actions included in each transaction.
```
Comment (lparisc) [inline docs/04_protocol/03_network_peer_protocol.md]: 224 bits of hash --> 224 bits of **the** hash

File: docs/04_protocol/03_network_peer_protocol.md (hunk 1 above)
Comment (lparisc) [reply docs/04_protocol/03_network_peer_protocol.md]: ah.. those inconspicuous spaces

File: docs/04_protocol/03_network_peer_protocol.md (hunk 2 above)
Comment (bobgt) [reply docs/04_protocol/03_network_peer_protocol.md]: Added **the** as per suggested.

**Pull request merged**